packages/airnode-deployer/src/utils/files.ts

```diff
@@ -32,11 +27,10 @@ export function writeReceiptFile(
       airnodeXpub: deriveAirnodeXpub(mnemonic),
     },
     deployment: {
-      airnodeAddressShort,
       cloudProvider: config.nodeSettings.cloudProvider as CloudProvider,
       stage: config.nodeSettings.stage,
       nodeVersion: config.nodeSettings.nodeVersion,
-      timestamp,
+      timestamp: new Date().toISOString(),
```

Comment:
I was thinking about that and didn't know what approach would be better. These two can vary by up to 15 min (GCP so slooow). I think the timestamp in the receipt should be of the time the deployment ended. But I guess that's up to interpretation. It can be the time it started :man_shrugging: But that's odd, isn't it?

Reply:
Won't users identify a particular deployment by that timestamp?

Reply:
You mean in the cloud? I don't think they will. First of all, they should not need to deal with the cloud resources manually. And second, the deployment is distinguished by Airnode address and stage. And you're currently deployed version is the one with the latest timestamp, no matter what's in your receipt. But I get what you're saying, I can sync those two timestamps, I don't have a strong preference.